alws/routers/builds.py

```diff
@@ -8,22 +8,23 @@
     status,
 )
 
-from alws import database
+from alws import database, models
+from alws.auth import get_current_user
 from alws.crud import (
     build as build_crud,
     build_node,
     platform as platform_crud,
     platform_flavors as flavors_crud
 )
-from alws.dependencies import get_db, JWTBearer
-from alws.errors import DataNotFoundError
+from alws.dependencies import get_db
+from alws.errors import DataNotFoundError, PermissionDenied
 from alws.schemas import build_schema
 
 
 router = APIRouter(
     prefix='/builds',
     tags=['builds'],
-    dependencies=[Depends(JWTBearer())]
+    dependencies=[Depends(get_current_user)]
```

Comment:
should we invoke `get_current_user` here like `JWTBearer`?

Reply:
Nope, it's ready to use without invocation